README.md

```diff
@@ -4,42 +4,42 @@ wpantund, Userspace WPAN Network Daemon
 `wpantund` is a user-space network interface driver/daemon that
 provides a native IPv6 network interface to a low-power wireless
 **Network Co-Processor** (or *NCP*). It was written and developed by
-Nest Labs to make supporting [Thread](http://threadgroup.org)
-connectivity on unix-like operating systems more straightforward.
+Nest to support [Thread](http://threadgroup.org)
+connectivity on uUnix-like operating systems and make the support more straightforward.
 
-`wpantund` is designed to marshall all access to the NCP, ensuring
-that it always remains in a consistent, well-defined state.
+`wpantund` is designed to marshall all access to the NCP ensuring
+that it always remains in a consistent and well-defined state.
 
 This is not an official Google product.
 
-## Feature and Architecture Summary ##
+## Feature and architecture summary ##
 
 `wpantund` provides:
 
- *  ...a native IPv6 interface to an NCP.
- *  ...a command line interface (`wpanctl`) for managing and
+ *  Native IPv6 interface to an NCP.
+ *  Command line interface (`wpanctl`) for managing and
     configuring the NCP.
- *  ...a DBus API for managing and configuring the NCP.
- *  ...a way to reliably manage the power state of the NCP.
- *  ...a uniform mechanism for handling NCP firmware updates.
+ *  DBus API for managing and configuring the NCP.
+ *  Method to reliably manage the power state of the NCP.
+ *  Uniform mechanism for handling NCP firmware updates.
 
 The architecture and design of `wpantund` has been motivated by the
-following design goals (in no particular order):
+following design goals (in no specific order):
 
- *  Portability across unix-like operating systems (Currently supports
+ *  Portability across Unix-like operating systems (currently supports
     Linux and OS X. BSD support should be fairly trivial to add)
- *  Require few runtime dependencies (DBus, with boost needed when
+ *  Requires few runtime dependencies (DBus, with boost needed when
```

Comment:
These are individual design goals, not a list of capabilities: It does not read correctly if you use plural.

Consider how the list reads if you use the prefix *"One of the design goals of `wpantund` is to"*:

* *"One of the design goals of `wpantund` is to"* Require few runtime dependencies

versus

* *"One of the design goals of `wpantund` is to"* Requires few runtime dependencies